### ISSUE TITLE
Enhance the shortcut to focus the terminal split panes

### ISSUE THREAD

**Issue by Cronos87** [warpdotdev/Warp#3219]
### Describe the solution you'd like?

```Text
I would like Warp to focus on the right terminal window when using the keyboard shortcut left and right.

Imagine I have a window that has 4 split terminals; if I focus on the upper left and press `cmd+option+right`, I will focus on the upper right; that is perfect, but if I press `cmd+option+left`, it will focus the bottom left terminal. It would be lovely to focus on the upper left one.

The same thing happens from the bottom left; if I want to focus on the bottom right, it will focus on the upper right.
```

### Is your feature request related to a problem? Please describe.

Here is a video that describes the "issue".

https://github.com/warpdotdev/Warp/assets/1923206/78d2708e-5f3a-46e9-b0ea-ff3cc541f541

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for this feature request @Cronos87! 

I've been able to replicate the behavior you described, it seems like when you go clockwise, the correct panes are selected, but counter-clockwise it's not as intuitive.

Please add a :+1: to the original post at the top to signal that you want this feature, and then hit the subscribe button if you'd like to be notified.

**Comment by Cronos87** [comment]
Yes, your way to explain this issue is much more elegant than mine.

I'm glad you understood what I wanted to describe.

Let me know if you need more information. I would be glad to help you 🙂

**Comment by Cronos87** [comment]
It works like a charm! Thank you very much!

**Comment by Cronos87** [comment]
I reopen this issue because it seems to have a regression. It doesn't work as intended; it appears to be broken with the latest versions.

**Comment by kevinchevalier** [comment]
Thank you.  I've made a fix that should be in the next release.

**Comment by Cronos87** [comment]
I confirm that after the new update, the issue is resolved and works as expected! Thank you again for the quick fix!

**Comment by kevinchevalier** [comment]
Great.  Closing this issue!
